ra/ra.go

```diff
@@ -379,7 +387,14 @@ func (ra *RegistrationAuthorityImpl) checkRegistrationIPLimit(ctx context.Contex
 		return err
 	}
 
-	if count.Count >= limit.GetThreshold(ip.String(), noRegistrationID) {
+	threshold, isOverridden := limit.GetThreshold(ip.String(), noRegistrationID)
+	if isOverridden && !ipRangeCheck {
+		// We do not support overrides for the NewRegistrationsPerIPRange limit.
```

Comment:
Then why do we need this extra logic at all? When `fuzzyRegLimit` is passed in as the `limit` parameter to this function, its `limit.GetThreshold()` is just always going to return `false` for `isOverridden` and that should be sufficient.

Reply:
🤦🏻‍♀️

Reply:
I unresolved this comment because this code block still exists. I'm of the opinion that the `NewRegistrationsPerIPRange` ratelimit `isOverridden` should be thrown away and the conditional removed.